communication/interface.go

```diff
@@ -17,55 +17,4 @@
 
 package communication
 
-import (
-	"github.com/mysteriumnetwork/node/identity"
-	"github.com/mysteriumnetwork/node/market"
-)
-
-// DialogWaiter defines server which:
-//   - waits and servers incoming dialog requests
-//   - negotiates with Dialog initiator
-//   - finally creates Dialog, when it is accepted
-type DialogWaiter interface {
-	Start() (market.Contact, error)
-	Stop() error
-	ServeDialogs(DialogHandler) error
-}
-
-// DialogHandler defines how to handle incoming Dialog
-type DialogHandler interface {
-	Handle(Dialog) error
-}
-
-// DialogEstablisher interface defines client which:
-//   - initiates Dialog requests to network
-//   - creates Dialog, when it is negotiated
-type DialogEstablisher interface {
-	EstablishDialog(peerID identity.Identity, peerContact market.Contact) (Dialog, error)
-}
-
-// Dialog represent established connection between 2 peers in network.
-// Enables bidirectional communication with another peer.
-type Dialog interface {
-	PeerID() identity.Identity
-	Sender
-	Receiver
-	Close() error
-}
-
-// Receiver represents interface for:
-//   - listening for asynchronous messages
-//   - listening and serving HTTP-like requests
-type Receiver interface {
-	Receive(consumer MessageConsumer) error
-	Respond(consumer RequestConsumer) error
-	Unsubscribe()
-}
-
-// Sender represents interface for:
-//   - sending asynchronous messages
-//   - sending and HTTP-like request and waiting for response
-type Sender interface {
-	Send(producer MessageProducer) error
-	Request(producer RequestProducer) (responsePtr interface{}, err error)
-}
+// TODO: remove this file when done.
```

Comment:
Looks like this file should be removed.

Reply:
Yes, for now I'm keeping it until this is PR is ready to merge because in case there are merge conflicts it would be easer to solve.